src/probnum/problems/zoo/diffeq/_ivp_examples.py

```diff
@@ -0,0 +1,186 @@
+import numpy as np
+
+from probnum.problems import InitialValueProblem
+
+__all__ = ["threebody", "vanderpol", "rigidbody"]
+
+
+def threebody(t0=0.0, tmax=17.0652165601579625588917206249, y0=None):
```

Comment:
this interface is "new", compared to the old threebody implementation before. This way one can just call `threebody()` which I found useful already.